<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcul frais transport</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Roboto Slab&quot;;
background-color: rgb(255, 255, 127);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>SOCIETE DE TRANSPORT EL AMEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtJ">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtM">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtA">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Expéditeur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtExp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDest">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Destinataire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>De</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDe">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Vers</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbVers">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Poid du Colis (Kg)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPoids">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
color: rgb(85, 170, 0);</string>
   </property>
   <property name="text">
    <string>Livraison Expess</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkExpress">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAjouter">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>300</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Roboto Slab&quot;;
background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblColis">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>1131</width>
     <height>341</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Expéditeur</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destinataire</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>De</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vers</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Poids</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Express</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Distance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prix</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
